### ISSUE TITLE
Monk Issues

### ISSUE THREAD

**Issue by Maqlovin** [FeenixServerProject/Pandaria_5.4.2_Issue_tracker#121]
The purpose of this post is listing all monk issues together.

!!!!I've given the issues a score based on importance of the bug, ranging from 1 (low importance (visual issues)) to 5 (absolutely needed to play monk properly)
## Spells
- [ ] Renewing Mist: Renewing mist should show with 3 charges on the first target, it should then jump to a new, injured (if possible) friendly target, showing 2 charges, it should then jump to a last target, currently it does not show the amount of charges remaining. It does jump to targets though. 2
- [ ] Soothing mist: Jade Serpent Statue should copy your soothing mist, while in combat, the statue doesn't seem to copy soothing mist, only out of combat. If the statue copies your soothing mist while out of combat, and you interrupt soothing mist, the statue keeps on channeling. 3
- [ ] Spear Hand Strike: Spear Hand Strike's silence component works, but the effect does not get affected by DRs. 2
- [ ] Surging mist: visual issue, if you cast it instantly with Soothing Mist and you don't move afterwards, which means, you just let the Soothing mist run out, you will visually still be trying to cast a surging mist. Also occurs doing the same with enveloping mist. 1
- [ ] Thunder Focus Tea: it doesn't show as being on cooldown, while it is and it gives a message 'mana is low' while it isn't if you use it. 2
- [ ] Transcendence: it doesn't leave a visible spirit of the caster behind to show where the teleport will bring the player. 2
- [ ] Jade Serpent Statue: the statue can currently be healed. 2
## Talents
- [ ] Chi Burst: Chi burst currently does nothing, it's a static ball doing no damage/healing. It should travel 40 yards, healing all friendly targets and dealing damage to all enemies in it's path. 5

Brewmaster

TO BE CONTINUED

**Comment by Zerixx** [comment]
" Monks don't have a starting weapon, and can't equip any types of weapons, they should be able to equip: staves, 1h sword/axe/fist/mace "

If you were playing Panda that was intended. Other races, we need to get that sorted.

**Comment by Zerixx** [comment]
@Maqlovin, not sure what Rev. you're on, but can you test like first 10-15 levels in all but human and NE zones, and or some systems please. These spell issues we can't fix until the DB is cleaned of all the corrupt data!

As always, much appreciated
-Z

**Comment by Zerixx** [comment]
As of https://github.com/ServerProject/Pandaria-Core/commit/33e46ed0bf6fa1b2ab26296aa8b2ad7632940fec, Monk and all other classes' spells now properly consume power, as well as do what they are intended to do in that aspect. Skills cost chi and chi's removed. Skills cost Burning Embers, and Burning Embers are removed. Simply put.

-Z

**Comment by Maqlovin** [comment]
Yeah, I was playing a human monk, I'm well aware a pandaren monk gets his weapon in one of his first quests, I will continue my testing as soon as the stances issue is resolved, after the monk spells&talents tests I will take a look at the starting zones

**Comment by Maqlovin** [comment]
@Zerixx The power consuming doesn't seem to be fixed for monks on all fields, spells still cost no energy when used in Stance of the fierce tiger as a mistweaver.

**Comment by ghost** [comment]
Tooltips fixed: https://github.com/ServerProject/Pandaria-Core/pull/96

**Comment by Zerixx** [comment]
Jab energy cost fixed in https://github.com/ServerProject/Pandaria-Core/commit/7676a40b1ba8a03f845974aee2bcdf2b5f853d2d
